lib/config.py

```diff
@@ -43,6 +43,9 @@
     ('--packages-repos-target-path', '-R'):
         dict(help='Directory where to clone code repositories',
              default='/var/lib/host-os/repositories'),
+    ('--no-update-packages-repos-before-build',):
+        dict(help='Update code repositories before building',
```

Comment:
This option actually does the opposite: "Do not update code repositories before building"